### DISCUSSION TITLE
Add react native expo to React Native Upgrade Helper

### DISCUSSION THREAD

**Discussion by adsalihac** [react-native-community/discussions-and-proposals#660] [Ideas]
react native cli and expo is more changes in native codebase. So is possible to add react native expo upgrader is better.

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. I think you should be posting this as a PR against the [React Native Upgrade Helper](https://github.com/react-native-community/upgrade-helper) that the code details of such an implementation could be discussed further.

That's the only way it can be discussed & reviewed with the maintainers.
